Nemt/nemt.cpp

```diff
@@ -9,8 +9,9 @@ Nemt::Nemt(int cl) {
 }
 
 inline int Nemt::convert_angle(int return_value) {
-    if (return_value < 404 && return_value > (-404)) {
-        return (return_value * 70 / 320) - 35;
+    if (return_value < NOT_FOUND && return_value > (-NOT_FOUND)) {
+        return (return_value * CAMERA_HORIZONTAL_ANGLE / CAMERA_WIDTH)
+                             - CAMERA_HORIZONTAL_ANGLE / 2;
```

Comment:
As far as I can see, this `- CAMERA_HORIZONTAL_ANGLE / 2` is not inside the parentheses, thus this should be aligned differently. Please put the minus (`-`) sign at the end of the previous line and align `CAMERA...` with `(` on previous line.